examples/OneOpenAir/OtaHandler.h

```diff
@@ -18,10 +21,23 @@ enum OtaUpdateOutcome {
 
 class OtaHandler {
 public:
-  void updateFirmwareIfOutdated(String deviceId) {
+  OtaHandler(StateMachine &sm, Configuration &config)
```

Comment:
Propose to use a callback instead of passing in the state machine. Also the deviceId should be sufficient to pass in, haven't checked really though.